### PR TITLE
Add image2video microservice (Stable Video Diffusion)

### PR DESCRIPTION
**Pull request by XinyuYe-Intel**
## Description

Add image2video microservice (Stable Video Diffusion).

## Issues

`n/a`.

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds new functionality)

## Dependencies

accelerate
diffusers
opencv-python
torch
transformers

## Tests

Describe the tests that you ran to verify your changes.

### CODE REVIEW

File: comps/image2video/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2024 Intel Corporation
```
Comment (chensuyue) [inline comps/image2video/Dockerfile:1]: Please add test for this new microservice.

Comment (XinyuYe-Intel) [reply comps/image2video/Dockerfile:1]: Added test.

**Review by lvliang-intel — APPROVED** [review]

**Comment by lkk12014402** [comment]
shall we merge the 2 microservices into 1? `[comps/image2video/svd/svd_server.py]` and `comps/image2video/image2video.py`

**Comment by XinyuYe-Intel** [comment]
> shall we merge the 2 microservices into 1? `[comps/image2video/svd/svd_server.py]` and `comps/image2video/image2video.py`

I tried it in 1 microservice, but put inference into microservice would cause some trouble, that's why I use fastAPI in svd service.

**Comment by lkk12014402** [comment]
> > shall we merge the 2 microservices into 1? `[comps/image2video/svd/svd_server.py]` and `comps/image2video/image2video.py`
> 
> I tried it in 1 microservice, but put inference into microservice would cause some trouble, that's why I use fastAPI in svd service.

let's sync offline.

**Comment by kevinintel** [comment]
please follow the folder structure as other comps

**Review by lkk12014402 — APPROVED** [review]

**Comment by XinyuYe-Intel** [comment]
> please follow the folder structure as other comps, we also need habana support

Changed folder structure and add habana support.

**Pull request merged**